webapp/src/Service/ImportExportService.php

```diff
@@ -83,15 +99,47 @@ public function getContestYamlData(Contest $contest, bool $includeProblems = tru
         return $data;
     }
 
+    protected function convertImportedTime(array $fields, array $data, ?string &$message = null): ?DateTimeImmutable
+    {
+        $timeValue = null;
+        $usedField = null;
+        foreach ($fields as $field) {
+            if ($timeValue) {
+                continue;
```

Comment:
Why not just break as soon as `$timeValue` gets set?

Reply:
Original author didn't do that, but makes sense IMO